manifests/config_aem.pp

```diff
@@ -2,14 +2,16 @@
   $run_mode,
   $tmp_dir,
   $aem_ssl_port,
+  $host,
   $aem_system_users           = undef,
   $aem_base                   = '/opt',
   $aem_id                     = 'aem',
   $aem_keystore_password      = undef,
   $aem_keystore_path          = undef,
   $cert_base_url              = undef,
   $enable_create_system_users = true,
-  $credentials_hash           = undef
+  $credentials_hash           = undef,
+  $aem_ssl_method             = undef,
```

Comment:
Probably best to define `jetty` as default parameter to keep backward compatibility.